lib/portlayer/storage/vsphere/datastore.go

```diff
@@ -15,9 +15,11 @@
 package vsphere
 
 import (
+	"fmt"
 	"io"
 	"net/url"
 	"path"
+	"regexp"
```

Comment:
@mavery helped me with the expression. Also, we know a Perl celebrity. True story.